moto/lexv2models/models.py

```diff
@@ -0,0 +1,397 @@
+    def describe_bot_alias(self, bot_alias_id: str, bot_id: str) -> Dict[str, Any]:
+        ba = self.bot_aliases[bot_alias_id]
+
+        return {
+            "botAliasId": ba.id,
+            "botAliasName": ba.bot_alias_name,
+            "description": ba.description,
+            "botVersion": ba.bot_version,
+            "botAliasLocaleSettings": ba.bot_alias_locale_settings,
+            "conversationLogSettings": ba.conversation_log_settings,
+            "sentimentAnalysisSettings": ba.sentiment_analysis_settings,
+            "botAliasHistoryEvents": ba.history_events,
+            "botAliasStatus": ba.status,
+            "botId": ba.bot_id,
+            "creationDateTime": ba.creation_date_time,
+            "lastUpdatedDateTime": ba.last_updated_date_time,
+            "parentBotNetworks": ba.parent_bot_networks,
+        }
+
+    def update_bot_alias(
+        self,
+        bot_alias_id: str,
+        bot_alias_name: str,
+        description: str,
+        bot_version: str,
+        bot_alias_locale_settings: Optional[Dict[str, Any]],
+        conversation_log_settings: Optional[Dict[str, Any]],
+        sentiment_analysis_settings: Optional[Dict[str, Any]],
+        bot_id: str,
+    ) -> Dict[str, Any]:
+        ba = self.bot_aliases[bot_alias_id]
+
+        ba.bot_alias_name = bot_alias_name
+        ba.description = description
+        ba.bot_version = bot_version
+        ba.bot_alias_locale_settings = bot_alias_locale_settings
+        ba.conversation_log_settings = conversation_log_settings
+        ba.sentiment_analysis_settings = sentiment_analysis_settings
+        ba.bot_id = bot_id
+        ba.last_updated_date_time = datetime.now().isoformat()
+
+        return {
+            "botAliasId": ba.id,
+            "botAliasName": ba.bot_alias_name,
+            "description": ba.description,
+            "botVersion": ba.bot_version,
+            "botAliasLocaleSettings": ba.bot_alias_locale_settings,
+            "conversationLogSettings": ba.conversation_log_settings,
+            "sentimentAnalysisSettings": ba.sentiment_analysis_settings,
+            "botAliasStatus": ba.status,
+            "botId": ba.bot_id,
+            "creationDateTime": ba.creation_date_time,
+            "lastUpdatedDateTime": ba.last_updated_date_time,
+        }
+
+    def list_bot_aliases(
+        self, bot_id: str, max_results: int, next_token: str
+    ) -> Tuple[List[Dict[str, Any]], Optional[str], str]:
+        bot_alias_summaries = [
+            {
+                "botAliasId": ba.id,
+                "botAliasName": ba.bot_alias_name,
+                "description": ba.description,
+                "botVersion": ba.bot_version,
+                "botAliasStatus": ba.status,
+                "creationDateTime": ba.creation_date_time,
+                "lastUpdatedDateTime": ba.last_updated_date_time,
+            }
+            for ba in self.bot_aliases.values()
+        ]
+
+        return bot_alias_summaries, next_token, bot_id
```

Comment:
We should not return the `next_token` that the user provides. Usually if a user wants to paginate this method, it will be a simple loop like:
```
while not next_token:
    # keep calling the same method
```
If we always return a value here, the loop will never end.

We should just return None here (or completely remove it from the code), to make it clear that pagination is not yet supported